### DISCUSSION TITLE
OpenCRVS - v1.1.0-stable

### DISCUSSION THREAD

**Discussion by euanmillar** [opencrvs/opencrvs-core#3986] [Technical Q&A]
We are proud to announce the next release of OpenCRVS **[v1.1.0-stable](https://documentation.opencrvs.org/general/releases/v1.1.0-stable-release-notes)** !  

This release contains important backup and security improvements, migration support and miscellaneous bug-fixes.  

All system administrators and implementers should begin the process of [migrating](https://documentation.opencrvs.org/general/releases/v1.0.1-to-v1.1.0-migration-notes).  

To learn more about our release process, check out our recent [blog post](https://www.opencrvs.org/resources/connect/blog/release-management-as-a-digital-public-good-what-we-have-learnt-so-far).  

Keep an eye on our incremental hot-fix releases [here](https://github.com/opencrvs/opencrvs-core/releases).  

Our next minor release v1.2.0-beta is scheduled for December 2022.

This discussion thread is dedicated to migrating from v1.0.1 to v1.1.0

**Comment by euanmillar** [comment]
Known issue ([will be resolved by this PR](https://github.com/opencrvs/opencrvs-farajaland/pull/359)):

Deployment script in the country configuration: 

We deprecated the `--update-metadata` parameter which is passed to `deploy.sh` from the Github [Deploy](https://github.com/o/zub8C4BetmW3a9Bj4Cd4/s/qI5B1zX9fZVx3HCjIY12/~/changes/RHqG2w9dnOSBqbwTiqva/setup/3.-installation/3.3-set-up-a-server-hosted-environment/3.3.6-deploy) Action.  The country configuration Github Action file `deploy.yml` still attempts to pass this parameter to `deploy.sh`.  This causes the deploy script to fail.  To resolve this issue, please edit the Github Action `deploy.yml `in your country configuration in lines 90, 112 and 134 from:

```
bash deploy.sh --clear-data=${{ env.FACTORY_RESET }} --restore-metadata=${{ env.FACTORY_RESET }} --update-metadata=no ...
```

to:
```
bash deploy.sh --clear-data=${{ env.FACTORY_RESET }} --restore-metadata=${{ env.FACTORY_RESET }} ...
```

**Comment by opencrvsuganda** [comment]
Howdy, I am having an issue with creating/registering a death event. I get the error "Date of death must be after deceased date of birth" even though the dates entered are okay. Unless there is a particular format that i am missing. I have attached the screenshots 
<img width="478" alt="Screenshot 2022-10-04 at 14 50 05" src="https://user-images.githubusercontent.com/114717732/193813031-71c9a85f-531f-49eb-acbd-3836abeadfea.png">
<img width="624" alt="Screenshot 2022-10-04 at 14 49 51" src="https://user-images.githubusercontent.com/114717732/193813046-b5578ece-ce35-405c-be80-56670163973c.png">
<img width="792" alt="Screenshot 2022-10-04 at 14 56 16" src="https://user-images.githubusercontent.com/114717732/193813454-2db9498d-17b3-404c-b686-1397250657a7.png">


Kindly double check the backend. 

Baldwin

**Comment by euanmillar** [comment]
Hey @opencrvsuganda looks like we cannot reproduce the issue you described with the date of birth. Perhaps the system date on the server was wrong at the time you were testing.  If you encounter any other potential bugs you would like us to take a look at, please raise an [issue](https://github.com/opencrvs/opencrvs-core/issues).
